src/constant_value.rs

```diff
@@ -23,10 +27,25 @@ pub enum ConstantValue {
         // todo: is there some way store the def_id here if available?
         // this would not be serialized/deserialized.
     },
-    // A place holder for other kinds of constants. Eventually this goes away.
+    /// Unsigned 16 byte integer.
+    U128(u128),
+    /// The Boolean true value.
+    True,
+    /// A place holder for other kinds of constants. Eventually this goes away.
     Unimplemented,
 }
 
+impl Into<AbstractValue> for ConstantValue {
```

Comment:
According to https://doc.rust-lang.org/std/convert/trait.Into.html, you should implement From instead of Into.